Revit_Core_Engine/Objects/WarningSwallower.cs

```diff
@@ -30,7 +32,9 @@ public class WarningSwallower : IFailuresPreprocessor
         /***************************************************/
         /****              Public methods               ****/
         /***************************************************/
-
+        [Description("Failure message swallower, used to continue the process without regard to errors.")]
```

Comment:
Your text should be the description of a class itself.

Here I would put something along the lines:
```suggestion
        
        [Description("Preprocess the failures, in this case swallow all of them.")]
```